go.mod

```diff
@@ -192,11 +191,11 @@ require (
 	kernel.org/pub/linux/libs/security/libcap/cap v1.2.76
 	sigs.k8s.io/hydrophone v0.7.0
 	sigs.k8s.io/yaml v1.5.0
+	source.monogon.dev v0.0.0-20250707205020-c4c89601ff80
```

Comment:
my concern with this one is that it's a monorepo, and it's quite huge, so we might have issues due to e.g. mixing up dependencies (like in this PR, e.g. nftables update), so we probably need a way to extract this library as a standalone repo

Reply:
if the license allows it - or we retain original copyright, but we do need to have something smaller than this